aksel.nav.no/website/playwright.config.ts

```diff
@@ -46,10 +46,11 @@ const config: PlaywrightTestConfig = {
         ...devices["Desktop Chrome"],
       },
       ...(process.env.FULL_TEST
```

Comment:
If we're going to have the same test suite run for both cases, we could also just remove this `FULL_TEST` env var.